GameDefinition/Content/ItemKindActor.hs

```diff
@@ -474,7 +474,7 @@ alligator = ItemKind
                , ("eye 6", COrgan), ("ear 9", COrgan)
                , ("animal brain", COrgan) ]
   }
-octopus = ItemKind
+giantOctopus = ItemKind
```

Comment:
BTW, the identifier needs to be mentioned at the start of the file in two places.